examples/triton/docker-compose-with-web.yml

```diff
@@ -0,0 +1,53 @@
+version: '2.1'
+
+services:
+  redis:
+    image: autopilotpattern/redis:3.2.8-r1.0.0
+    mem_limit: 4g
+    restart: always
+    # expose for linking, but each container gets a private IP for
+    # internal use as well
+    expose:
+      - 6379
+      - 26379
+    labels:
+      - triton.cns.services=redis
+    environment:
+      - CONTAINERPILOT=file:///etc/containerpilot.json
+      - affinity:com.docker.compose.service!=~redis
+    env_file: _env
+    network_mode: bridge
+
+  consul:
+    image: autopilotpattern/consul:0.7.2-r0.8
+    command: >
+      /usr/local/bin/containerpilot
+      /bin/consul agent -server
+        -bootstrap-expect 1
+        -config-dir=/etc/consul
+        -ui-dir /ui
+    restart: always
+    mem_limit: 128m
+    ports:
+      - 8500
+    expose:
+      - 53
+      - 8300
+      - 8301
+      - 8302
+      - 8400
+      - 8500
+    labels:
+      - triton.cns.services=redis-consul
+    network_mode: bridge
+
+  webserver:
+    image: jasonpincin/ap-redis-example-webserver:1.0.0
```

Comment:
I can publish this as `autopilotpattern/ap-redis-example-webserver:1.0.0`, was just publishing under my scope while iterating on the blog post and testing.